spec/features/visitors/sign_up_spec.rb

```diff
@@ -255,19 +242,6 @@
     expect(current_path).to eq user_confirmation_path
   end
 
-  context 'confirmation instructions sent to existing user', email: true do
-    xit 'does not send an email to the existing user' do
```

Comment:
@pkarman caught up with Ryan. Apparently there *is* a resend link in the registrations page:

![screen shot 2016-10-28 at 10 58 02 am](https://cloud.githubusercontent.com/assets/601515/19816756/6f3dec0c-9cfd-11e6-9f51-749bf55818fd.png)

So that comment from the previous PR doesn't really make sense. Updated and un-pending'd the test for the resend functionality during sign up since that is not exercised in the specs anywhere else.